src/app/drive/services/downloadManager.service.ts

```diff
@@ -300,73 +327,96 @@ export class DownloadManagerService {
     };
 
     const addFileStreamToZip = async (index: number, driveItem: DownloadItemType) => {
-      let fileStream: ReadableStream<Uint8Array>;
-      const lruFilesCacheManager = await LRUFilesCacheManager.getInstance();
-      const cachedFile = await lruFilesCacheManager.get(driveItem.id.toString());
-      const isCachedFileOlder = checkIfCachedSourceIsOlder({ cachedFile, file: driveItem });
-
-      if (cachedFile?.source && !isCachedFileOlder) {
-        const blob = cachedFile.source;
-        downloadProgress[index] = 1;
-        fileStream = blob.stream();
-      } else {
-        const downloadedFileStream = await downloadFile({
-          fileId: (driveItem as DriveFileData).fileId,
-          bucketId: (driveItem as DriveFileData).bucket,
-          creds: {
-            user: (driveItem as AdvancedSharedItem).credentials?.networkUser ?? credentials.credentials.user,
-            pass: (driveItem as AdvancedSharedItem).credentials?.networkPass ?? credentials.credentials.pass,
-          },
-          mnemonic: (driveItem as AdvancedSharedItem).credentials?.mnemonic ?? credentials.mnemonic,
-          options: {
-            abortController,
-            notifyProgress: (totalBytes, downloadedBytes) => {
-              const progress = downloadedBytes / totalBytes;
+      // TODO: QA REMOVE
+      if (driveItem.name.includes('internxt_test_file') || driveItem.plainName?.includes('internxt_test_file')) {
+        failedItems.push(driveItem);
+        return;
+      }
```

Comment:
Perfect, I left a check to remember to remove before merge

- [ ] Removed forced error for QA